src/MPoly.jl

```diff
@@ -1467,7 +1467,7 @@ function ConformanceTests.generate_element(Rx::MPolyRing)
   len_bound = 8
   exp_bound = rand(1:5)
   len = rand(0:len_bound)
-  coeffs = [ConformanceTests.generate_element(R) for _ in 1:len]
+  coeffs = elem_type(R)[ConformanceTests.generate_element(R) for _ in 1:len]
```

Comment:
In case of empty lists, this was of type `Vector{Any}` despite the generation function being type stable. This caused more problems down the road.